<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="20190719_142430" timestamp="2019-07-19T14:24:30" time="18.014" tests="11" errors="2">
  <testsuite id="1" name="exampleProject" package="exampleProject" hostname="ATMECSINDT-043" timestamp="2019-07-19T14:24:31" time="9.186" tests="11" errors="2">
    <properties>
      <property name="Run From" value="ATMECSINDT-043" />
      <property name="Run OS" value="amd64 Windows 10 10.0" />
      <property name="Run User" value="deepankar.pathal" />
      <property name="Iteration" value="1 of 1" />
      <property name="Script" value="exampleProject.xlsx" />
      <property name="Logs" value="C:\projects\exampleProject\output\20190719_142430\logs\nexial-20190719_142430.log" />
      <property name="Data File" value="exampleProject.data.xlsx" />
      <property name="DataSheet(s)" value="#default" />
      <property name="JAVA_OPT" value="-Dorg.apache.poi.util.POILogger=org.apache.poi.util.NullLogger" />
      <property name="runtime args" value="-script C:\projects\exampleProject\artifact\script\exampleProject.xlsx" />
    </properties>
    <testcase id="Activity 1" name="Activity 1" classname="Scenario" timestamp="2019-07-19T14:24:32" time="7.431">
      <failure type="failure found in test activity" message="2 of 11 FAILED. Check C:\projects\exampleProject\output\20190719_142430\exampleProject.20190719_142431.001.xlsx for execution result, C:\projects\exampleProject\output\20190719_142430\logs\nexial-20190719_142430.log for execution logs" />
      <system-out>Total : 11
Passed: 9
Failed: 2
Output: C:\projects\exampleProject\output\20190719_142430\exampleProject.20190719_142431.001.xlsx
Logs  : C:\projects\exampleProject\output\20190719_142430\logs\nexial-20190719_142430.log</system-out>
    </testcase>
  </testsuite>
</testsuites>
